app/views/last_seen_issues_plugin/_update_last_seen_issues.html.erb

```diff
@@ -0,0 +1,34 @@
+<%
+  issues_display_limit = LastSeenIssuesPluginSettings.get_setting(:number_for_last_displayed_issues_setting)
```

Comment:
See the comment above about code duplication

Reply:
Done.